source/administration/changelog.md

```diff
@@ -4,11 +4,233 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+- Pinning a post in center channel no longer changes pinned posts list in the right-hand sidebar.
+- Pinning a post in center now adds the badge to search results.
+- Fixed error message text for **Edit URL** field in channel creation modal.
+- Disabled config file watcher while running from makefile.
+- Fixed Go client's `GetTeamByName()` function.
+- Recent mentions search now properly includes `@[username]` in the search.
+- Updated error message when entering a password longer than maximum number of characters.
+- Don't send the same message multiple times when hitting "Retry" on a failed post.
+- Deleting your own pinned post in a direct message channel now properly disappear from the pinned posts list.
+- Fixed channel purpose help text for private channels.
+- When ability to change the header is restricted, "Set a Header" option is no longer shown in the channel intro.
+- Mention notifications now trigger if the word is formatted in bold, italic or strikethrough.
+- Mention notifications now longer trigger if the word is inside a code block.
+- In mobile view, Manage Members menu option no longer reads "View Members" for channel admins.
+- Fixed vertical space between posts in the right-hand sidebar.
+- Fixed flagged posts and recent mentions icon sizing in the header.
+- Fixed vertical space below link previews.
+- Usernames with dots now get mention notifications when followed by a comma or other symbol.
+- Deactivated users are no longer listed in the "Manage Members" modal.
+- Collapsable Account Setting menus now properly open in iOS Safari and Chrome browsers.
+- Removing an expired license now removes the blue bar header message.
+- "Next" button in More Channels list now takes you to the top of the next page, instead of the bottom.
+- Blue bar "Preview Mode" header message now disappears after enabling email notifications.
+- Full name is now editable in Account Settings if the first and last name attributes are not specified in **System Console > Authentication > LDAP**.
+- Added a back button to pinned posts list on the right-hand sidebar.
+- "Pinned" icon no longer overlaps text on consecutive posts or replies that hvae Markdown headings.
+- Uploading a profile picture on iOS no longer throws an error.
+- Fixed group message names in channel switcher (CTRL/CMD+K) for group messages not in your sidebar.
+- Channel notification preferences no longer appear saved when clicking Cancel.
+
+### Compatibility  
+
+#### Breaking changes:
+
+- If you're using NGINX as a proxy for the Mattermost Server, replace the `location /api/v3/users/websocket {` line with `location ~ /api/v[0-9]+/(users/)?websocket$ {` in the `/etc/nginx/sites-available/mattermost` NGINX configuration file. [See documentation to learn more](https://docs.mattermost.com/install/install-ubuntu-1404.html#configuring-nginx-as-a-proxy-for-mattermost-server).
```

Comment:
It might be needed for current and future requirements, but does it break previous installations? If not, it shouldn't be in the Breaking Changes section.